docs/dev/new_check_howto.md

```diff
@@ -274,6 +274,72 @@ The check is almost done. Let's add the final touches.
 
 ## Final touches
 
+### Enhance the configuration file
```

Comment:
When reading this, it seems this paragraph is optional. 
I don't think we should make people think it is optional.